Essentials/src/main/java/com/earth2me/essentials/commands/Commandpay.java

```diff
@@ -14,10 +14,15 @@
 import java.util.Collections;
 import java.util.List;
 import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 import static com.earth2me.essentials.I18n.tl;
 
 public class Commandpay extends EssentialsLoopCommand {
+    private static final Pattern NUMBER_FORMAT = Pattern.compile("([0-9][0-9_'`,]*(?:\\.[0-9]+)?|\\.[0-9]+)([kmbt])?");
+    private static final Pattern SANITIZE = Pattern.compile("[^0-9.]");
```

Comment:
Need to confirm whether we ever supported commas as a decimal separator via the config options in the past. I don't *think* we did, but would like to be sure first

Reply:
Well, the current behavior is that `1,000` is currently being interpreted as `1000`, so all I did was try to keep it this way.

Reply:
Yeah, that's going to be since anything aside from numbers and decimal places are ignored by the SANITIZE regex (which was just unnamed before). Comma was not specifically designated as a separator however (and there is another open issue for that, as based on server locale, technically `1,000.23` should be as equally valid as `1.000,23` for example). Doesn't necessarily need to be a concern for this PR though as long as the existing behavior is maintained.